Nimb3s.Automaton.DataAccess/JobData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Nimb3s.Automaton.Pocos;
+using Nimb3s.Automaton.Pocos.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Nimb3s.Automaton.DataAccess
+{
+    public class JobData
+    {
+        private readonly string _connectionString;
+        private SqlDataAccess db = new SqlDataAccess();
+
+        public JobData()
+        {
+            _connectionString = "Data Source=AAVELDANEZ-PC;Initial Catalog=Automaton;Integrated Security=True;Connect Timeout=60;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
+        }
+
+        public JobStatusModel GetJobStatusById(Guid jobId)
+        {
+            string sql = @"SELECT j.JobName, jst.Enumeration
```

Comment:
this definitely does the job but it exposes us to a [SQL injection attack](https://portswigger.net/web-security/sql-injection). should use a stored procedure like mentioned above